spec/src/main/asciidoc/WebServicesMetadata.adoc

```diff
@@ -127,63 +128,64 @@ This specification addresses the need to simplify:
 enable the creation of every Web Service that it is possible to write
-within existing specifications (JAX-WS 2.0 [5] provides finer control
+within existing specifications (Jakarta XML Web Services 2.3 [5] provides finer control
 over the resultant Web Service). The goal is to make it easy to build
 the most common types of Web Services.
 
-== 2 Concepts
+== Concepts
 
 This section summarizes the following concepts and processes of the
-JSR-181 specification:
+Jakarta Web Services Metadata specification:
 
-* Programming model for JSR-181 Web Services
-* Use of metadata in JSR-181
+* Programming model for Jakarta Web Services Metadata
+* Use of metadata in Jakarta Web Services
 * Non-normative processing model for a JWS file
-* Runtime requirements for a JSR-181 container
+* Runtime requirements for a Jakarta Web Services Metadata container
 * Annotations used for WSDL, binding and configuration
 
 The metadata is formally described in section 4.
 
-=== 2.1 Programming Model Overview
+=== Programming Model Overview
 
-JSR-181, along with JAX-WS and JSR-109, defines a programming model for
+Jakarta Web Services Metadata, along with Jakarta XML Web Services and 
+Jakarta Enterprise Web Services, defines a programming model for
 building a Web Service. A developer who builds a Web Service with these
 technologies is required to write and manage several artifacts: a WSDL
 document describing the external Web Service contract; a service
 endpoint interface defining the Java representation of the Web Service
 interface; a service implementation bean containing the Web Service
 implementation; and one or more deployment descriptors linking the WSDL,
-interface, and implementation into a single artifact. JSR-181 simplifies
+interface, and implementation into a single artifact. Jakarta Web Services Metadata simplifies
 this model by allowing the developer to write only the service
 implementation bean - _actual business logic_ – and use annotations to
 generate the remaining artifacts.
 
-=== 2.2 Development Models
+=== Development Models
 
-JSR-181 defines several different models of Web Service development.
+Jakarta Web Services Metadata defines several different models of Web Service development.
 Only the Start with Java development model is REQUIRED by
 implementations.
 
-== 2.2.1 Start with Java
+==== Start with Java
 
 Following the “Start with Java” development model, the developer begins
 by writing a Java class to expose as a Web Service. The developer then
-runs this Java class through the JSR-181 processor, which produces WSDL,
+runs this Java class through the Jakarta Web Services Metadata processor, which produces WSDL,
 schema, and other deployment artifacts from the annotated Java code. By
 default, the WSDL produced from the Java source follows the Java to
-XML/WSDL mapping defined by JAX-WS 2.0. However, the developer may
+XML/WSDL mapping defined by Jakarta XML Web Services  2.3. However, the developer may
```

Comment:
`Jakarta XML Web Services  Specification` instead of `Jakarta XML Web Services  2.3.`

Reply:
ok and other places too (3d54eee)